avm/res/network/azure-firewall/main.bicep

```diff
@@ -14,7 +14,7 @@ param name string
 param azureSkuTier string = 'Standard'
 
 @description('Conditional. Shared services Virtual Network resource ID. The virtual network ID containing AzureFirewallSubnet. If a Public IP is not provided, then the Public IP that is created as part of this module will be applied with the subnet provided in this variable. Required if `virtualHubId` is empty.')
-param virtualNetworkId string = ''
+param virtualNetworkResourceId string = ''
```

Comment:
Although I agree with the change, updating an input parameter is a breaking change. For azure fw, since the module was not published, I guess we can avoid to care. But for bastion and vnet this would mean a minor version update.

Reply:
Hence I updated the version number...